cloud-optimized-geotiffs/intro.qmd

```diff
@@ -1,48 +1,37 @@
 
 If you plan to pan and zoom a large amount of data through a tiling service in a web browser, there is a tradeoff between 1 large file, or many smaller files. The current recommendation is to meet somewhere in the middle, a moderate amount of medium files.
 
-### Internal Blocks
+## Internal Blocks
+
+> This attribute is also sometimes called **chunks** or **internal tiles**.
 
 Internal blocks are required if the dimensions of data are over 512x512. However you can control the size of the internal blocks. 256x256 or 512x512 are recommended. When displaying data at full resolution, or doing partial reading of data this size will impact the number of reads required. A size of 256 will take less time to read, and read less data outside the desired bounding box, however for reading large parts of a file, it may take more total read requests. Some clients will aggregate neighboring block reads to reduce the total number of requests.
 
-### Overviews
+## Overviews
 
 Overviews are downsampled (aggregated) data intended for visualization.
 The best resampling algorithm depends on the range, type, and distribution of the data.
 
 The smallest size overview should match the tiling components’ fetch size, typically 256x256. Due to aspect ratio variation just aim to have at least one dimension at or slightly less than 256. The COG driver in GDAL, or rio cogeo tools should do this.
 
-There are many resampling algorithms for generating overviews. When creating overviews several options should be compared before deciding which resampling method to apply.
-
-## How to create and validate COGs
-
-1. [Rio-cogeo: GitHub - cogeotiff/rio-cogeo: Cloud Optimized GeoTIFF creation and validation plugin for rasterio](https://github.com/cogeotiff/rio-cogeo)
-2. [Gdal: COG – Cloud Optimized GeoTIFF generator — GDAL documentation](https://gdal.org/drivers/raster/cog.html)
-
-
-## Additional Resources
-
-* [Planet Blog: An Introduction to Cloud Optimized GeoTIFFS (COGs) Part 1: Overview](https://developers.planet.com/docs/planetschool/an-introduction-to-cloud-optimized-geotiffs-cogs-part-1-overview/)
-* [COG Talk — Part 1: What’s new?](https://medium.com/devseed/cog-talk-part-1-whats-new-941facbcd3d1)
-* [Development Seed Blog: Do you really want people using your data?](https://medium.com/@_VincentS_/do-you-really-want-people-using-your-data-ec94cd94dc3f)
+There are many [resampling algorithms](https://gdal.org/programs/gdal_translate.html#cmdoption-gdal_translate-r) for generating overviews. When creating overviews several options should be compared before deciding which resampling method to apply.
 
-## How to visualize COGs
+## See more
 
-* GDAL vis* drivers (vsicurl, vsis3, vsiaz,)
-* Titiler https://github.com/developmentseed/titiler
-* Rio-viz https://github.com/developmentseed/rio-viz
+- Additional [GeoTIFF details](cogs-details.qmd) that can be helpful.
```

Comment:
Even if these details are in fact useful for all GeoTIFFs, given that in this context we're talking solely about COG, would it be less confusing to the user to say "COG details" instead of "GeoTIFF details"?